src/console/content.ts

```diff
@@ -533,6 +538,24 @@ class ConsoleContent extends Widget {
     this.prompt.mimetype = this._mimetype;
   }
 
+  protected addContentCell(cell: CodeCellWidget) {
+    this._content.addWidget(cell);
+    const prompt = this.prompt;
+    const index = prompt ? this._cells.indexOf(prompt.model) : -1;
```

Comment:
I don't understand the logic here. It seems as though `prompt` will always be defined.